### PR TITLE
Orders backend orders api routes #55

### PR DESCRIPTION
**Pull request by Zezlita**
#55 
Created patch & delete API routes

### CODE REVIEW

File: routes/orders.js [hunk 1]
```diff
@@ -50,4 +54,50 @@ const {
...
+        } catch({name, message}){
+            next({
+                name:"updatedOrderMessage",
+                message:"Great News! Your order status has been updated!"
+            })}
+        })
+
+
+//=====DELETE /orders/:orderId (**):Update the order's status to cancelled
+    ordersRouter.delete('/:orders/:orderId', requireUser, async (req, res, next) => {
```
Comment (tpadilla10117) [inline routes/orders.js:82]: Quick fix: '/orders/:orderId'

**Review by tpadilla10117 — CHANGES_REQUESTED** [review]
Looks good! One small fix in the comments.  We'll see if the others spot anything else!  Also need to test these ASAP, unless you did already!

File: routes/orders.js (hunk 1 above)
Comment (Zezlita) [reply routes/orders.js]: Thanks Trin! This is fixed!

**Review by tpadilla10117 — APPROVED** [review]

File: routes/orders.js [hunk 2]
```diff
@@ -50,4 +54,51 @@ const {
...
+
+
+//=====DELETE /orders/:orderId (**):Update the order's status to cancelled
+    ordersRouter.delete('/orders/:orderId', requireUser, async (req, res, next) => {
+        const {getOrderById} = req.params;
+
+        try{
+            const order = await getOrderById(id);
+
+            if(order && order.userId !== req.userId){
```
Comment (kati-unu) [inline routes/orders.js:89]: req.userId works? vs req.user.id?

File: routes/orders.js [hunk 3]
```diff
@@ -50,4 +54,51 @@ const {
     } )
 
 
+//=====PATCH /orders/:orderId (**):Update an order, notably change status
+    ordersRouter.patch('/orders:orderId', requireUser, async (req, res, next) => {
```
Comment (kati-unu) [inline routes/orders.js:58]: remove /orders for both routes

Comment (kati-unu) [reply routes/orders.js:58]: and add route to the index.js

**Review by kati-unu — CHANGES_REQUESTED** [review]

File: routes/orders.js (hunk 2 above)
Comment (Zezlita) [reply routes/orders.js]: adjusted to req.user.id

File: routes/orders.js (hunk 3 above)
Comment (Zezlita) [reply routes/orders.js]: updated!

**Review by kati-unu — APPROVED** [review]

**Pull request merged**